config/routes.rb

```diff
@@ -8,7 +8,7 @@
       :via => Devise.mappings[:user].sign_out_via
   end
 
-  resources :vendors, only: [:index, :show]
+  resources :vendors, param: :folio_id, only: [:index, :show]
```

Comment:
The `VendorsController` was already using `folio_id` but the route was previously expecting an `id`. This fixes that. Only found this because I added vendor views.

Reply:
thanks!  I wondered if there was another spot for this.

Reply:
I just learned about it today 😄